### PR TITLE
Move to JAVA 11 in Docker File

### PR DESCRIPTION
**Pull request by superpoussin22**
### What was a problem?

```
Java 11 is now the LTS version for Java , so the dockerfile need to be updated
```

### How this PR fixes the problem?

```
It add Java 11 in the image
```

### Check lists (check `x` in `[ ]` of list items)

- [X] Run MobSF unit tests (http://your-mobsf-@ip:8000/tests/ or python3 manage.py test)
- [X] Tested Working on Linux, Mac, and Windows and Docker
- [X] Coding style (indentation, etc)

### Additional Comments (if any)

```
None
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -29,12 +29,19 @@ RUN apt update -y && apt install -y \
     libxml2-dev \
     libxslt1-dev
 
-#Install Oracle JDK 8
```
Comment (ajinabraham) [inline Dockerfile]: Please remove unnecessary comments

File: Dockerfile [hunk 2]
```diff
@@ -91,6 +98,9 @@ RUN git clone --recursive ${YARA_URL} yara-python && \
     rm -fr /tmp/yara-python && \
     sed -i 's/APKID_ENABLED.*/APKID_ENABLED = True/' /root/Mobile-Security-Framework-MobSF/MobSF/settings.py
 
+# Set Java 11 Directory
+RUN sed -i 's#JAVA_DIRECTORY = ""#JAVA_DIRECTORY = "/usr/bin/"#' /root/Mobile-Security-Framework-MobSF/MobSF/settings.py
```
Comment (ajinabraham) [inline Dockerfile]: Is this required. Wouldn't Java 11 be the default. Instead of this. Make sure Java 11 is the default.

Comment (ajinabraham) [reply Dockerfile]: You should investigate why it's failing.

File: Dockerfile (hunk 1 above)
Comment (superpoussin22) [reply Dockerfile]: done

File: Dockerfile (hunk 2 above)
Comment (superpoussin22) [reply Dockerfile]: it come's from the fact that script in profile.d are not launched , so the jdk.sh script which set the ENV is not sourced , shell in docker is not interactive

Comment (ajinabraham) [reply Dockerfile]: @superpoussin22 Can we avoid setting Java directory as in line 102?

Comment (superpoussin22) [reply Dockerfile]: @ajinabraham yep done 
I modify utils.py cause /usr/bin/python did note provide Oracle in output 
let me know if ok for you

**Pull request merged**